typescript/sdk/src/token/app.ts

```diff
@@ -42,12 +35,12 @@ export class HypERC20App extends GasRouterApp<
   }
 
   static fromAddressesMap(
-    addressesMap: HyperlaneAddressesMap<HypERC20Factories & ProxiedFactories>,
+    addressesMap: HyperlaneAddressesMap<HypERC20Factories>,
     multiProvider: MultiProvider,
   ): HypERC20App {
     const helper = appFromAddressesMapHelper(
       addressesMap,
-      { ...hypERC20factories, ...proxiedFactories },
```

Comment:
No longer passing proxied factories as we cannot assume that we own the relevant `proxyAdmins` and that we have the addresses i.e we don't have the renzo warp route router proxyAdmin addresses in config